ansible/roles/configure/files/helpers/conv_acl.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
It was used to convert ACL rules list from the xml format to the json file for the swssconfig
If we ever decide to update the list of the rules (from the different xml) we will need this tool